### DISCUSSION TITLE
Allow slots to have attributes

### DISCUSSION THREAD

**Discussion by JanMalte** [wrabit/django-cotton#224] [Announcements]
Allow slots to have attributes, so you can have more control over them.

```html
<!-- usage -->
<c-accordion class="something">
  <c-slot name="title" class="something-else">
  </c-slot>
</c-accordion>
```

```html
<!-- cotton/accordion.html -->
<div class="{{ class }}">
  <div class="{{ title.class }}">
    {{ title }}
  </div>
</div>
```

_Originally posted by @wrabit in https://github.com/wrabit/django-cotton/issues/70#issuecomment-2241521662_

**Comment by wrabit** [comment]
The reasoning for this is not so clear at the moment, it would be good to see some more real world need for this ability. 

I’m hesitant to throw it in until the additional processing is justified, so moving to discussion to allow more people to come forward to voice their opinions.
